dbt_integration.py

```diff
@@ -168,6 +169,26 @@ def wrapper(
 
     return wrapper
 
+def default_profiles_dir(project_dir) -> Path:
+    if "DBT_PROFILES_DIR" in os.environ:
+        return Path(os.environ["DBT_PROFILES_DIR"]).resolve()
+    return project_dir if (project_dir / "profiles.yml").exists() else Path.home() / ".dbt"
```

Comment:
os.path.join() would make it is agnostic

Reply:
it is agnostic already as we use vscode.Uri later on.